src/cmd/nsfs.js

```diff
@@ -71,13 +71,14 @@ Arguments:
 const OPTIONS = `
 Options:
 
-    --http_port <port>         (default 6001)   Set the S3 endpoint listening HTTP port to serve.
-    --https_port <port>        (default 6443)   Set the S3 endpoint listening HTTPS port to serve.
-    --https_port_sts <port>    (default -1)     Set the S3 endpoint listening HTTPS port for STS.
-    --https_port_iam <port>    (default -1)     Set the endpoint listening HTTPS port for IAM.
-    --metrics_port <port>      (default -1)     Set the metrics listening port for prometheus.
-    --forks <n>                (default none)   Forks spread incoming requests (config.ENDPOINT_FORKS used if flag is not provided).
-    --debug <level>            (default 0)      Increase debug level.
+    --http_port <port>              (default 6001)   Set the S3 endpoint listening HTTP port to serve.
+    --https_port <port>             (default 6443)   Set the S3 endpoint listening HTTPS port to serve.
+    --https_port_sts <port>         (default -1)     Set the S3 endpoint listening HTTPS port for STS.
+    --https_port_iam <port>         (default -1)     Set the endpoint listening HTTPS port for IAM.
+    --metrics_port       <port>     (default -1)     Set the metrics listening port for prometheus.
+    --metrics_https_port <port>     (default -1)     Set the metrics listening https port for prometheus.
```

Comment:
1. make sure HTTPS is capital letters like the other HTTPS usage
2. why default is -1?